Sources/App/Core/Search.swift

```diff
@@ -76,11 +76,9 @@ enum Search {
     static func query(_ database: Database,
```

Comment:
`query` used to create the actual query object and run the fetch. It's now to separate functions `query` and `fetch` so that we can render and test the generated SQL from `query`. Just a split and rename.